youtube_transcript_api/_api.py

```diff
@@ -100,12 +105,16 @@ class _TranscriptFetcher():
     API_BASE_URL = 'https://www.youtube.com/api/{api_url}'
     LANGUAGE_REGEX = re.compile(r'(&lang=.*&)|(&lang=.*)')
 
-    def __init__(self, video_id, languages):
+    def __init__(self, video_id, languages, proxy):
         self.video_id = video_id
         self.languages = languages
+        self.proxy = proxy
```

Comment:
thinking about it, `proxies` would probably be the more appropriate name, as it is also used for requests.